src/ProgramSpeedController.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import React from 'react';
+import { Form } from 'react-bootstrap';
+import type { IntlShape } from 'react-intl';
+import { injectIntl } from 'react-intl';
+import { ReactComponent as SlowIcon } from './svg/Slow.svg';
+import { ReactComponent as FastIcon } from './svg/Fast.svg';
+import './ProgramSpeedController.scss';
+
+type ProgramSpeedControllerProps = {
```

Comment:
Please add a TODO comments:

    // TODO: The Form.Control in configured for exactly 5 steps but the values
    //       array param can be any length. Update the code to either constrain
    //       the values array to 5 elements, or to calculate the number of steps
    //       for the slider from the values array.